src/MemberContainer.js

```diff
@@ -47,7 +47,7 @@ const Member = ({member, isOdd, shape}) => (
     flexDirection={direction(isOdd)}
   >
     <MemberQuestions member={member} />
-    <FlexItem mb={[6, 8, 8, 0, 0]} flexShrink="0" style={{position: 'relative'}}>
+    <FlexItem mb={[6, 8, 8, 0, 0]} css={{flexShrink: 0}} style={{position: 'relative'}}>
```

Comment:
does it matter that we have `css=` and `style=` here? Do they do the same thing?

Reply:
One adds the CSS via emotion and one adds it inline 🤔 We probably should pick one or the other